File: TypeScript/codewars/Typescript/Drying-Potatoes.ts
export function potatoes(p0: number, w0: number, p1: number): number {
	const dryMatter = (w0 * (100 - p0)) / 100;
	const w1 = dryMatter / (1 - p1 / 100);
	return Math.floor(w1);
}

File: Csharp/CodeWars/src/Completed/DryingPotatoes.cs
namespace CodeWars.Completed;

public class DryingPotatoes
{
    public static int Potatoes(int p0, int w0, int p1)
    {
        var dryMatter = w0 * (100 - p0);

        return (int)Math.Floor((decimal)dryMatter / (100 - p1));
    }
}
